Loop/Managers/AlertPermissionsChecker.swift

```diff
@@ -98,97 +104,9 @@ fileprivate extension AlertPermissionsChecker {
             listeningToNotificationCenter = true
         }
     }
-    
-    private func notificationCenterSettingsChanged(_ newValue: NotificationCenterSettingsFlags) {
-        if !issueOrRetract(alert: Self.riskMitigatingAlert,
-                           condition: newValue.requiresRiskMitigation,
-                           alreadyIssued: UserDefaults.standard.hasIssuedRiskMitigatingAlert,
-                           setAlreadyIssued: {UserDefaults.standard.hasIssuedRiskMitigatingAlert = $0}) {
-            _ = issueOrRetract(alert: Self.scheduledDeliveryEnabledAlert,
-                               condition: newValue.scheduledDeliveryEnabled,
-                               alreadyIssued: UserDefaults.standard.hasIssuedScheduledDeliveryEnabledAlert,
-                               setAlreadyIssued: {UserDefaults.standard.hasIssuedScheduledDeliveryEnabledAlert = $0})
-        }
-    }
-    
-    private func issueOrRetract(alert: LoopKit.Alert, condition: Bool, alreadyIssued: Bool, setAlreadyIssued: (Bool) -> Void) -> Bool {
-        if condition {
-            if !alreadyIssued {
-                alertManager?.issueAlert(alert)
-                setAlreadyIssued(true)
-            }
-            return true
-        } else {
-            if alreadyIssued {
-                setAlreadyIssued(false)
-                alertManager?.retractAlert(identifier: alert.identifier)
-            }
-            return false
-        }
-    }
-}
-
-fileprivate extension AlertPermissionsChecker {
```

Comment:
Moving the definition of these particular alerts to AlertManager seems like a step backwards in terms of separation of responsibilities. 

I think it makes sense for this class to delegate to AlertManager for issuance, retraction, and general management of these alerts, but the definition/content of the alerts themselves is nice to keep close to the logic that determines the conditions.  Like we have DeviceManagers define the alert content and evaluate the conditions under which to issue/retract, but they delegate to something else for issuance/retraction.

I know the AlertManager *does* have some alert definitions, and some condition evaluation logic for Loop not Looping alerts and Bluetooth disabled alerts, but I feel the same about those; it'd probably be cleaner (and more testable) if those were split out as well.  In this case, we already have a "Checker" class split out, so it feels right to keep the alert definition and condition logic in this class. The seam between this class and the AlertManager could then just be the `AlertIssuer` protocol?